src/table/components/TableBodyWrapper.jsx

```diff
@@ -25,6 +25,8 @@ function TableBodyWrapper({
   const hoverEffect = layout.components?.[0]?.content?.hoverEffect;
   const bodyStyle = useMemo(() => getBodyStyle(layout, theme), [layout, theme.name()]);
 
+  // active: turn off interactions that affect the state of the visual representation including selection, zoom, scroll, etc.
+  // select: turn off selections.
   const selectionsEnabled = !!selectionsAPI && !constraints.active;
```

Comment:
Bit now you don't check constraints.select?

Reply:
wrong commit before. Fixed. Thanks.